indico_themes_canonical/static/css/_default/_forms.css

```diff
@@ -40,6 +40,11 @@ input[type]:not([type="number"]):not([type="checkbox"]):not([type="radio"]):not(
   width: 100%;
 }
 
+// Override default error input bg color width default Vanilla
+.ui.form .field.error {
+  background-color: #f8e9e8;
```

Comment:
Would recommend setting a `color` as well, we've had a few situations where not doing so bit us.